Library/Homebrew/os/linux/elf.rb

```diff
@@ -126,12 +134,31 @@ def needed_libraries_using_patchelf_rb(path)
     end
   end
 
+  def save_using_patchelf(new_interpreter, new_rpath)
+    patchelf = DevelopmentTools.locate "patchelf"
+    odie "Could not locate patchelf, please: brew install patchelf." if patchelf.nil?
+    Homebrew.safe_system(
+      patchelf,
+      "--set-interpreter", new_interpreter,
+      "--force-rpath", "--set-rpath", new_rpath,
+      to_s
+    )
+  end
+
+  def save_using_patchelf_rb(new_interpreter, new_rpath)
+    patcher = patchelf_patcher
+    patcher.interpreter = new_interpreter if new_interpreter
+    patcher.rpath = new_rpath if new_rpath
```

Comment:
```suggestion
    patcher.rpath = new_rpath
```
I don't believe `new_rpath` can ever be `nil`.

Reply:
it can be nil when `interpreter` is changed but there was no rpath,  i believe we have binaries with no no rpath, otherwise why do we have safety guards like `if old_rpath`